README.md

```diff
@@ -65,11 +65,12 @@ account. Note that authenticating to WordPress.com via Google is not supported i
 
 Note that credentials created with our [WordPress.com applications manager][5] allow login only and not signup. New
 accounts must be created using the [official app][1] or [on the web](https://wordpress.com/start). Login is restricted
-to the WordPress.com account with which the credentials were created. Also, you will be able to interact with sites of
-that same WordPress.com account only. In other words, if the credentials were created with foo@email.com, you will only
-be able to login with foo@email.com and access foo@email.com sites. Using another account like bar@email.com will cause
+to the WordPress.com account with which the credentials were created. In other words, if the credentials were created with foo@email.com, you will only
+be able to login with foo@email.com. Using another account like bar@email.com will cause
 the `Client cannot use "password" grant_type` error. 
 
+**Note**: Due to a known [issue](https://github.com/wordpress-mobile/WordPress-Android/issues/8754), login with a wordpress account with 2FA enabled is not currently fully supported on _development builds_.  If you _must_ login with an account with 2FA enabled on development builds, please follow instructions listed in the workaround [here](DEV-BUILD-2FA-LOGIN-ISSUE.md).
```

Comment:
This isn't completely accurate - there are server-side restrictions to using 2FA accounts with personal OAuth client credentials that would remain even after we resolve the issue to deal with the failed account settings fetch during login more gracefully.

I would change this to say something like:

> For security reasons, some account-related actions aren't supported for development builds when using a WordPress.com account with 2-factor authentication enabled. There is also currently an issue where a restart of the app is required to complete login in this case: [issue]